### PR TITLE
More Block doc improvements

### PR DESCRIPTION
**Pull request by Hixie**
Mostly around pointing people towards Block more aggressively.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/scrollable.dart [hunk 1]
```diff
@@ -916,11 +916,11 @@ class ScrollNotification extends Notification {
 ///
 /// See also:
 ///
+///  * [Block], if your single child is a [BlockBody] or a [Column].
```
Comment (abarth) [inline packages/flutter/lib/src/widgets/scrollable.dart]: BlockBody seems a bit obscure to mention here.

**Comment by abarth** [comment]
LGTM

File: packages/flutter/lib/src/widgets/scrollable.dart (hunk 1 above)
Comment (Hixie) [inline packages/flutter/lib/src/widgets/scrollable.dart]: updated

**Comment by Hixie** [comment]
will check in after travis clears it

**Pull request merged**